.styleguidist/styleguidist.sections.js

```diff
@@ -167,6 +168,12 @@ module.exports = {
                 'LanguageMenuLink/LanguageMenuLink',
               ]),
             },
+            {
+              name: 'Notification',
+              components: getComponentWithVariants('Notification')([
```

Comment:
the `getComponentWithVariants` method seems to generate an unnecessary subheading on the page now that the component has no variants presented on the same page.

Reply:
Fixed. Now function call is getComponents.